course/auth.py

```diff
@@ -680,9 +774,35 @@ def user_profile(request):
 
     user_form = None
 
+    def has_verified_inst_id(user):
+        if not EDITABLE_INST_ID_PRE_VRF:
+            return True and user.institutional_id or False
+        else:
+            if user.institutional_id_verified:
+                return True
+            else:
+                return False
+
+    def enable_inst_input(request):
```

Comment:
It's fine to do it on the server end as long as the naming makes it clear that it's not intended to be about rule enforcement logic but rather just UI convenience.